gspread/utils.py

```diff
@@ -894,17 +910,20 @@ def is_full_a1_notation(range_name: str) -> bool:
     return A1_ADDR_FULL_RE.search(range_name) is not None
 
 
-def get_a1_from_absolute_range(range_name: str) -> str:
+def get_a1_from_absolute_range(range_name: Optional[str]) -> str:
     """Get the A1 notation from an absolute range name.
     "Sheet1!A1:B2" -> "A1:B2"
     "A1:B2" -> "A1:B2"
+    None -> ""
 
     Args:
-        range_name (str): The range name to check.
+        range_name (str | None): The range name to check.
 
     Returns:
         str: The A1 notation of the range name stripped of the sheet.
     """
+    if range_name is None:
+        return ""
```

Comment:
I would recommend not ot change that function that is used by other worksheet method a bit everywhere. and instead pass a range name or skip calling it as this is made to handle a range name and return some value if we don't have a range name then we don't call that function.

In my opinion

Reply:
I agree